src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashApiClient.java

```diff
@@ -436,6 +451,91 @@ public String call() throws Exception {
     return response;
   }
 
+  private String putRequest(String path, String status) throws UnsupportedEncodingException {
```

Comment:
if the UnsupportedEncodingException comes from requesting "UTF-8" it's rather unlikely. Please catch it inside the metod and log (not printStackTrace)